components/Navbar.tsx

```diff
@@ -321,23 +329,29 @@ export const Navbar: React.FC = () => {
                     {section.groups.map(group => (
                       <div className="py-2" key={group.name}>
                         {group.routes.map(route => (
-                          <Link key={route.href} href={route.href} className="block text-white text-xl py-6 px-6 mr-6 hover:bg-sunglow-400 hover:text-black active:bg-sunglow-200">
-                            {route.name}
-                          </Link>
+                          route.href === "https://publications.ccv.brown.edu" ? (
```

Comment:
same here... i wouldn't just be checking if it matches the publications link (string not even a variable!!!) 

In the route config have like an internal/external key for true/false???